hummingbot/market/liquid/liquid_auth.py

```diff
@@ -21,7 +21,7 @@ def generate_signature(self, path_url: str) -> (Dict[str, Any]):
         """
         auth_payload = {
             'path': path_url,
-            'nonce': str(time.time()),
+            'nonce': str(time.time() * 1e3),
```

Comment:
just realized what both before and after will have decimals.
e.g. 
old ts: `1581314372.6992035`
new ts: `1581314372699.209`

How about let's remove the decimal points similar to the suggested format from the API doc by casting the expression to int, something along this line? `str(int(time.time() * 1e3))`, what do you think? @paleboot

Reply:
reference: https://developers.liquid.com/#authentication
`str(int(time.time() * 1e3))` => `1581314372699`

Reply:
Hi, thanks for checking out my PR! That's actually a great idea, totally missed on that one. I made the change, please do check it out!